j_SpringJDBCTemplateExample/pom.xml

```diff
@@ -33,7 +33,7 @@
 		<dependency>
 		    <groupId>mysql</groupId>
 		    <artifactId>mysql-connector-java</artifactId>
-		    <version>5.1.48</version>
+		    <version>8.0.16</version>
```

Comment:
It is better to have the 5.1 version as the higher (or the latest 8.0) version gives some error for timezone.
I was not able to fix it so I am using the earlier MySQL connector version.